RocketRL/python/func/data_processing.py

```diff
@@ -69,10 +104,10 @@ def my_formula(x,m,c):
     return m*x +c
 
 def save_plot(fig,title):
-    save_dir = os.path.join(CWD_PATH, cfg['result_path'])
+    save_dir = os.path.join(cfg['CWD_PATH'], cfg['result_path'],cfg['fig_path'])
     fig.savefig(save_dir + title+".png")
 
-def plot_3var(cfg,df):
+def plot_3dvar(cfg,df):
     #plots 3 input variables in 3d, with output variable as the color of the points
     #for 3 output variables, this will create 3 plots.
     #user manually adjusts which 3/4 input variables to plot
```

Comment:
Functions descriptions should be docstrings instead of comments.

In this case it would be:
```python
def plot_3dvar(cfg,df):
    """
    Plots 3 input variavles in 3d, with output variavles as the color of the points.
    For 3 output variables, this will create 3 plots.
    User manually adjusts which 3/4 input variables to plot.
    """
```